### DISCUSSION TITLE
mysql 执行sum时，结果返回{{}}

### DISCUSSION THREAD

**Discussion by iwifigame** [cloudwu/skynet#1394] [Ideas]
当使用mysql，执行sum语句时，如下，如果结果是空，返回的结果是{{}}。使用起来不是很方便。
看能不能改成如果结果是空，返回和其它SELECT语句一样的结果{}。
```
SELECT SUM(Column_1)
FROM Table
WHERE Column_2 = 'Test'
```

**Comment by cloudwu** [comment]
我不太用 mysql 。但从代码看 https://github.com/cloudwu/skynet/blob/master/lualib/skynet/db/mysql.lua#L614

返回 {} 还是 {{}} 是服务器回应的数据决定的，并非 driver 的行为。

**Reply by iwifigame** [reply]
是服务器回应决定的。看能不能在driver层处理一下。返回一样的结果
